### PR TITLE
Updated table view image for listing management. 

### PR DESCRIPTION
**Pull request by stephaniesaylor**

### CODE REVIEW

**Comment by wski** [comment]
Will merge once table view is merged w/ `ozp-center`

**Comment by stephanieerin** [comment]
:+1:

**Comment by clarkjefcoat** [comment]
@wski - Table view has been merged.  Can we accept this PR now?

**Comment by wski** [comment]
This has merge conflicts which must be resolved first

**Comment by stephaniesaylor** [comment]
I think I have the conflicts taken care of, will someone please review this to see if it can be merged? Thank you.

File: src/scripts/components/articles/Center_listingManagementPage_orgSteward.html [hunk 1]
```diff
@@ -41,8 +41,58 @@
...
+
+<img border="0" alt="Table Actions" src="Doc_images/ListingManagementTable_ActionIcons.png" />
+
+<h3>Use the Listing Management Table</h3>
+<p>From the table view, stewards can:</p>
+<ul>
+	<li>Preview a listing by clicking its Name. From here, you can publish pending listings using the Administration tab.</li>
+	<li>See the listing owner&#39;s profile by clicking their name.</li>
+	<li>Read the last entry in the listing&#39;s change log (entered in the comments field).</li>
+<<<<<<< HEAD
```
Comment (blaino) [inline src/scripts/components/articles/Center_listingManagementPage_orgSteward.html]: Maybe I'm looking at an old commit, but if I'm not, this conflict needs to be fixed.

Comment (stephaniesaylor) [inline src/scripts/components/articles/Center_listingManagementPage_orgSteward.html]: OK, i'll check it out, thanks.

Comment (stephaniesaylor) [inline src/scripts/components/articles/Center_listingManagementPage_orgSteward.html]: OK, I think I got it. Please check it out. Thank you!

**Comment by blaino** [comment]
:+1:

**Comment by jal-vsi** [comment]
Passed testing in Listing environment using IE11, FF24, and CH36.  This fix only applies to Org Stewards and Admins.  Both roles can select Listing Management, then select "All Center Listings" (for Admin) or "Steward's Listings" (for Org Stewards), then select 'Table View' icon to observe the additional view.

**Pull request merged**